www/src/Locales/en/PinMapping.jsx

```diff
@@ -3,8 +3,10 @@ export default {
 	'sub-header-text':
 		'Use the form below to reconfigure your button-to-pin mapping.',
 	'alert-text':
-		"Mapping buttons to pins that aren't connected or available can leave the device in non-functional state. To clear the the invalid configuration go to the <1>Reset Settings</1> page.",
+		"Mapping buttons to pins that aren't connected or available can leave the device in non-functional state. To clear the invalid configuration go to the <1>Reset Settings</1> page.",
 	'pin-header-label': 'Pin',
+	'profile-pins-warning':
+		'Try to avoid changing the buttons/directions used for your switch profile hotkeys, or else it will get hard to understand what profile you are selecting!',
```

Comment:
Grammar change:

Try to avoid changing the buttons and/or directions used for the switch profile hotkeys. Otherwise, it will be difficult to understand what profile is being selected!

Reply:
Awesome, thanks! Will change!